### PR TITLE
v1.25 Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+
+### Bug Fixes
+ - Fixed an issue where Mattermost monokai theme no longer worked properly on mobile apps.
+ - Fixed an issue on Android where the notification badge count didn't update when using multiple channels.
+ - Fixed an issue on Android where test notifications did not work properly.
+ - Fixed an issue where "In-app" notifications caused the app badge count to get out of sync.
+ - Fixed an issue on Android where email notification setting displayed was not updated when the setting was changed.
+ - Fixed an issue where Favorite channels list didn't update if the app was running in the background.
+ - Fixed an issue where the timezone setting did not update when changing it back to set automatically.
+ - Fixed an issue where accessing Mobile tab reversed auto-reply setting to OFF when auto-reply was set to ON.
```
Comment (esethna) [inline CHANGELOG.md]: > accessing Mobile tab

Not sure what this one means

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+ - Fixed an issue on iOS where clicking on a hashtag from "recent mentions" (or flagged posts) returned the user to the channel instead of displaying hashtag search results.
+ - Fixed an issue where tapping on a hashtag engaged a keyboard for a moment before displaying search results.
+ - Fixed an issue where posts of the same thread appeared to be from different threads if separated by a new message line.
+ - Fixed an issue on iOS where the Name, Purpose and Header information appeared “boxed in” on channel info screen.
```
Comment (esethna) [inline CHANGELOG.md]: > appeared “boxed in” on channel info screen.

In cases like this we can probably just say we fixed styling in the specified area.

**Comment by amyblais** [comment]
@enahum Can you help merge this so it's available for customers? I can open a separate PR for any other edits where needed.

**Pull request merged**